src/containers/add-attachments/AddAttachmets.tsx

```diff
@@ -0,0 +1,108 @@
+import { ChangeEvent, FC, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { useModalContext } from '~/context/modal-context'
+import { ajustColumns } from '~/utils/helper-functions'
+import useBreakpoints from '~/hooks/use-breakpoints'
+import useSort from '~/hooks/table/use-sort'
+
+import { Box } from '@mui/material'
+import Typography from '@mui/material/Typography'
+import SearchIcon from '@mui/icons-material/Search'
+
+import { styles } from '~/containers/add-attachments/AddAttachments.styles'
+import {
+  initialSort,
+  columns,
+  removeColumnRules
+} from '~/containers/add-attachments/AddAttachments.constants'
+
+import InputWithIcon from '~/components/input-with-icon/InputWithIcon'
+import AppButton from '~/components/app-button/AppButton'
+import EnhancedTable from '~/components/enhanced-table/EnhancedTable'
+import { Attachment } from '~/types'
+
+const mockItems = [
+  {
```

Comment:
You can move out mock data to a constant file

Reply:
done